cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -177,17 +177,19 @@ function Multiselect( element ) { // eslint-disable-line max-statements, inline-
       } );
 
       var inputData = {
-        'id':        option.value,
+        'id':     option.value,
         // Type must come before value or IE fails
-        'type':    'checkbox',
-        'value':   option.value,
-        'name':    _name,
-        'class':   'cf-input cf-multi-select_checkbox',
-        'inside':  li
-      }
+        'type':   'checkbox',
+        'value':  option.value,
+        'name':   _name,
+        'class':  'cf-input cf-multi-select_checkbox',
+        'inside': li
```

Comment:
Keys don't actually need quoting unless they're hyphenated (they just need to be consistently quoted or not), so not sure if the quotes are wanted here or not.